### DISCUSSION TITLE
Index Lifecycle

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#12281] [Unsupported Versions]
Good Morning,

Zeek logs take up the most space in my Elasticsearch indexes.  Currently I am trying to delete after 9 days. ( I am waiting for more storage)

I have it set in the Grid configuration to 9 days, but after applying the changes, it still shows the default 365 days in elasticsearch.

Do I have to delete the settings in the Global Override area before it will take effect?

**Comment by cm-ops** [comment]
Changing the index setting in SOC worked on my test machine:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63416267/bb3c5438-2d34-46c5-b998-dd0dabbb754a)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63416267/2a040a0e-ab8e-4d60-b7bb-2824aff60b71)

**Reply by oneCrazyAdmin** [reply]
What is you setting in the global override?  Here are mine and it says that they are default.
![2024-01-31_09-30](https://github.com/Security-Onion-Solutions/securityonion/assets/78620004/ef64e9d8-47a7-4de5-85bc-d6b62bc424ca)

Also, what was the query that you ran to display the index settings from the command line?

**Reply by cm-ops** [reply]
I did not touch global_overrides when I change the setting for Zeek. To see changes to ES from the CLI, run `sudo salt-call pillar.get elasticsearch`
